draft-ietf-capport-api.md

```diff
@@ -99,16 +100,18 @@ If the client is unable to validate the certificate presented by the API server,
 
 ## JSON Keys  {#json-keys}
 
-The Captive Portal API data structures are specified in JavaScript Object Notation (JSON) [RFC8259]. Requests and responses for the Captive Portal API use the "application/captive+json" media type. Clients SHOULD include this media type as an Accept header in their GET requests, and servers MUST mark this media type as their Content-Type header in responses.
+The Captive Portal API data structures are specified in JavaScript Object Notation (JSON) {{RFC8259}}. Requests and responses for the Captive Portal API use the "application/captive+json" media type. Clients SHOULD include this media type as an Accept header in their GET requests, and servers MUST mark this media type as their Content-Type header in responses.
 
 The following keys are defined at the top-level of the JSON structure returned by the API server:
 
 - "captive" (required, boolean): indicates whether the client is in a state of captivity, i.e it has not satisfied the conditions to access the external network. If the client is captive (i.e. captive=true), it can still be allowed enough access for it to perform server authentication {{server-auth}}.
-- "user-portal-url" (required, string): provides the URL of a web portal with which a user can interact.
+- "user-portal-url" (optional, string): provides the URL of a web portal with which a user can interact.
```

Comment:
are we sure that the user-poral-url should be made "optional" ?

Reply:
As @martinthomson had mentioned in his issue for this, it might make the portal a bit useless for now, but if we're thinking about the future with extensibility, locking ourselves into this key for all use cases doesn't seem like a great idea.